modules/icingaweb2/templates/kickstart.ini.erb

```diff
@@ -1,5 +1,5 @@
 [config]
-endpoint = icinga2
+endpoint = <%= @fqdn %>
```

Comment:
Should be a separate commit next time, at first glance it wasn't clear to me why it is needed here.

Reply:
This was replaced with the puppet-icingaweb2 and module handling.

Reply:
This was replaced with the puppet-icingaweb2 and module handling.